homeassistant/components/schluter/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""The Schluter DITRA-HEAT integration."""
+from datetime import timedelta
+import logging
+
+from requests import RequestException, Session
+from schluter.api import Api
+from schluter.authenticator import AuthenticationState, Authenticator
+import voluptuous as vol
+
+from homeassistant.const import CONF_PASSWORD, CONF_TIMEOUT, CONF_USERNAME
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+DATA_SCHLUTER_SESSION = "schluter_session"
+DATA_SCHLUTER_API = "schluter_api"
+PLATFORMS = ["climate"]
```

Comment:
Do we get more platforms in the future? If not, we can remove this and just forward it to that single platform directly.

Reply:
My initial intention was to make it extensible to other platforms, but realistically I think climate is likely to be the only platform based on Schluter's product line. I will make this change